code/style.css

```diff
@@ -6,39 +6,58 @@ body {
   margin: 0;
   padding: 0;
   font-family: 'Montserrat', sans-serif;
-  background: #0026ff;
+  background-image:url(https://images.unsplash.com/photo-1525362081669-2b476bb628c3?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1234&q=80);
+}
+.overlay::before{
```

Comment:
What is read before the overlay(::before)? Before the bot appears? looks professional but cant figure out what it does :)